### ISSUE TITLE
There are no tests

### ISSUE THREAD

**Issue by Vaguery** [luminus-framework/luminus-migrations#1]
An example of how to actually call the `migrate` function with the `rollback` and `migrate` arguments and some individual migration numbers would be useful.

That information does not appear to be in the docs, so both testing migrations and testing most db functions is _extremely_ frustrating.

**Comment by yogthos** [comment]
I do have migrations documented on [the framework site](http://www.luminusweb.net/docs/migrations.md), but I'll add them in the readme as well.

**Comment by Vaguery** [comment]
Yes, I'm actually talking about the inability to _test_ migrations without an example showing how to do so.

**Comment by yogthos** [comment]
Testing is not really in the scope for the library, can you give an example of what specifically you'd like to be able to do.
